server/src/routes/api/v1/squidsRouter.js

```diff
@@ -0,0 +1,16 @@
+import express from "express"
+
+import Squid from "../../../models/Squid.js"
+import { nextWrapper } from "../../lib/nextWrapper.js"
+
+export const squidsRouter = new express.Router()
+
+squidsRouter.get(
+  "/",
+  nextWrapper(async (req, res) => {
```

Comment:
nice! I bet you'll tackle this in the pagination story, but it's a good idea to always constrain our index routes. At a minimium, I'll set a provisional limit to a high number like 1000 or 5000 to eliminate the possibility of the server running out of memory with a large record count.